fvm/src/syscalls/actor.rs

```diff
@@ -112,3 +113,11 @@ pub fn install_actor(
     let typ = context.memory.read_cid(typ_off)?;
     context.kernel.install_actor(typ)
 }
+
+pub fn actor_balance(
+    _context: Context<'_, impl Kernel>,
+    _addr_off: u32,
+    _addr_len: u32,
```

Comment:
I'd consider:

```suggestion
    actor_id: u64,
```

This will require an extra syscall to lookup the actor ID in some cases, but it matches our other syscalls. All the calls that take an _address_ do so because the target actor may not exist (yet).

Reply:
ok, will use the actor id.

Reply:
done.